user_docs/en/userGuide.t2t

```diff
@@ -1934,6 +1934,19 @@ This slider allows you to set the volume of NVDA sounds and beeps.
 This setting only takes effect when "Volume of NVDA sounds follows voice volume" is disabled.
 This option is not available if you have started NVDA with [WASAPI disabled for audio output #WASAPI] in Advanced Settings.
 
+==== Duration of keeping audio device awake ====[KeepAudioAwakeDuration]

```

Comment:
Personally, in English, I find this title slightly awkward.
Here is a suggested alternative. Note that I didn't change the anchor name, as that is referenced else ware, and should be changed holistically if it is changed.
I'm not in love with this alternative either, but it may be somewhat better.

```suggestion
==== Keep audio device awake time ====[KeepAudioAwakeDuration]
```

Reply:
I think I would just go with `Time to keep audio device awake`

Reply:
agreed with Sascha here